### PR TITLE
all: Remove S3 bucket management

### LINKED ISSUE #25: [3] Move S3 bucket management to separate repository
Managing S3 buckets doesn't make much sense for standalone clusters without the apps. We should move this to a separate project and make it part of the wrapper instead of creating them as a part of the cluster deployment process.

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:
To remove S3 bucket management from this module.

**Which issue this PR fixes**:
fixes #25 

**Special notes for reviewer**:
I have not done any special tests other than to init a new config repo and set up a new cluster from it.

I had to do a lot of manual terraform changes as the pre-commit test in the pipeline failed on the Terraform fmt step. However, I could not reproduce it on my local environment as it always passed when I ran `pre-commit run --show-diff-on-failure --color=always --all-files`. Terraform fmt did not want to update any of my files either so I updated them manually until the pipeline tests passed.

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/ck8s-cluster/blob/master/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

File: terraform/citycloud/modules/cluster/variables.tf [hunk 1]
```diff
@@ -7,29 +7,29 @@ variable "ssh_pub_key" {
   type        = string
 }
 
-variable public_ingress_cidr_whitelist {
-  type = list
+variable "public_ingress_cidr_whitelist" {
+  type = list(any)
```
Comment (lentzi90) [inline terraform/citycloud/modules/cluster/variables.tf]: These should probably be `list(string)`.

File: terraform/citycloud/variables.tf [hunk 2]
```diff
@@ -58,24 +58,24 @@ variable "master_anti_affinity_policy_wc" {
   type        = string
 }
 
-variable public_ingress_cidr_whitelist {
-  type = list
+variable "public_ingress_cidr_whitelist" {
+  type = list(any)
```
Comment (lentzi90) [inline terraform/citycloud/variables.tf]: `list(string)`

File: terraform/modules/openstack/secgroups/variables.tf [hunk 3]
```diff
@@ -2,14 +2,14 @@ variable "prefix" {
   type = string
 }
 
-variable public_ingress_cidr_whitelist {
-  type = list
+variable "public_ingress_cidr_whitelist" {
+  type = list(any)
```
Comment (lentzi90) [inline terraform/modules/openstack/secgroups/variables.tf]:
```suggestion
  type = list(string)
```

File: terraform/safespring/modules/cluster/variables.tf [hunk 4]
```diff
@@ -7,29 +7,29 @@ variable "ssh_pub_key" {
   type        = string
 }
 
-variable public_ingress_cidr_whitelist {
-  type = list
+variable "public_ingress_cidr_whitelist" {
+  type = list(any)
```
Comment (lentzi90) [inline terraform/safespring/modules/cluster/variables.tf]:
```suggestion
  type = list(string)
```

File: terraform/safespring/variables.tf [hunk 5]
```diff
@@ -58,24 +58,24 @@ variable "master_anti_affinity_policy_wc" {
   type        = string
 }
 
-variable public_ingress_cidr_whitelist {
-  type = list
+variable "public_ingress_cidr_whitelist" {
+  type = list(any)
```
Comment (lentzi90) [inline terraform/safespring/variables.tf]:
```suggestion
  type = list(string)
```

**Review by lentzi90 — COMMENTED** [review]
Really nice to get rid of this!

I think the `type` for the whitelists should be `list(string)`, otherwise LGTM. :slightly_smiling_face:

**Comment by OlleLarsson** [comment]
> I think the `type` for the whitelists should be `list(string)`, otherwise LGTM.

Oki, I've now updated this :)

**Review by lentzi90 — APPROVED** [review]

**Review by viktor-f — APPROVED** [review]
LGTM!

**Pull request merged**